README.md

```diff
@@ -78,6 +78,17 @@ In this section you will:
 
 You are now in the Learner portal for the enterpriser customer `Test Enterprise`, as a learner!
 
+## Next up: enroll one or more learners into your own course!
+
+It is very useful to create multiple courses and enroll the learners into them for testing
+
+For this, an easy way is to use the Studio! Usually at http://localhost:18010
+
+* Once there, create a course using the Create Course button
+* Once course is created, click 'View Live' to see the courese
+* Grab the course id from the url which will be similar to: http://localhost:18010/course/course-v1:testinguniversity+cs111111+summer2020
+  * In this case the course id is `course-v1:testinguniversity+cs111111+summer2020`
+* Now use the `Manage learners` page described in an earlier section, to enroll one or more learners into this course!
```

Comment:
err which section is this referring to? There isn't a section that describes adding a learner to a course in the Manage Learners page. I know it may seem like it's for dummies, but I'd probably specify that it is in the Link Learners section on the right of the Manage Learners page.

Also don't know if I mentioned it previously, but given that the "Manage Learners" button is way over on the far right at the top, that might be something to actually mention when you do first mention the Manage Learner button. Maybe I'm a weirdo, but my eyes follow the data, so I just end up scrolling all the way to the bottom on the left looking for this button, before I scroll back up and realize "oh it's on the right". Probably fine on tiny monitors, but on a widescreen.... :p

Reply:
added a link to a prior section since I already described this piece there, hth